app/validators/validate-response-format.ts

```diff
@@ -0,0 +1,34 @@
+import ContributorModel from 'ember-osf-web/models/contributor';
+
+function isEmptyString(checkString: string) {
```

Comment:
you could go with `isEmpty` util from `@ember/utils`.

Reply:
I didn't even see the `isEmpty` suggestion, but that does everything I want it to do.